tools/image-info

```diff
@@ -301,6 +301,20 @@ def read_fstab(tree):
     return result
 
 
+def read_sysconfig(tree):
+    result = {
+        "kernel": [],
+        "network": []
+    }
+    with contextlib.suppress(FileNotFoundError):
+        with open(f"{tree}/etc/sysconfig/kernel") as f:
+            result["kernel"] = f.read().splitlines()
+        with open(f"{tree}/etc/sysconfig/network") as f:
+            result["network"] = f.read().splitlines()
```

Comment:
If you put both open calls in just one `with` block then you wan't attempt to read the second file in case the first one don't exists.
Also maybe parse the actual key, value pairs. You should be able to re-use the code we use in osbuild to parse [`os-release`](https://github.com/osbuild/osbuild/blob/main/osbuild/util/osrelease.py#L17).
I think I would also not pre-fill the result dict so you don't get a lot of basically empty entries when the files don't exist.

Reply:
Updated this to use the parsing done in `os-release`

Reply:
Not a blocker, but good practice (and sometimes required for copyright) is to mention where the `os-release` code is coming from (in the commit message or as a comment in the code)